src/items.cpp

```diff
@@ -1205,12 +1191,12 @@ bool dropItem(Item* item, int player, bool notifyMessage)
 
 	if ( multiplayer == CLIENT )
 	{
-		strcpy((char*)net_packet->data, "DROP");
```

Comment:
I'd rather leave as c-style casts for all the network packet code due to quantity of changes will be a lot

Reply:
Fixed in 7b658c6c